build_tools/benchmarks/common/benchmark_definition.py

```diff
@@ -311,33 +313,44 @@ class BenchmarkInfo:
   run_config_id: Optional[str] = None
 
   def __str__(self):
+    return self.benchmark_name
+
+  @classmethod
+  def build_with_legacy_name(cls, model_name: str, model_tags: Sequence[str],
```

Comment:
Could you add a docstring to this method explaining what it does and why its exists?

Reply:
Done.

Reply:
Wait this docstring is basically just a restatement of the method name :-) Useful info I think is that this name is based on attempting to combine all the benchmark characteristics, new benchmarks are given human-defined names, the name is still used as a primary key in the legacy benchmark system, etc